webpack.config.js

```diff
@@ -69,7 +70,7 @@ module.exports = {
 
     optimization: {
         minimize: production,
-        minimizer: [`...`, new CssMinimizerPlugin()],
+        minimizer: [new TerserJSPlugin(), new CssMinimizerPlugin()],
```

Comment:
This actually should be the default according to https://webpack.js.org/configuration/optimization/#optimizationminimizer 
At least that what I understood when I reviewed the initial PR.

Reply:
So I just tried it now locally, and indeed it does not work without so I am happy with the fix. Too bad that the webpack docs are a but fuzzy (to me at least).